include/libtorrent/aux_/ffs.hpp

```diff
@@ -48,6 +48,17 @@ namespace libtorrent { namespace aux
 	// this function statically determines if hardware or software is used
 	// and expect the range to be in big-endian byte order
 	TORRENT_EXTRA_EXPORT int clz(span<std::uint32_t const> buf);
+
+	// these functions expect the range to be in big-endian byte order
+	// find last zero (flz)
+	TORRENT_EXTRA_EXPORT int flz_sw(span<std::uint32_t const> buf);
```

Comment:
I'm not super excited about the name "flz". Not clz either to be honest, but at least there's some wider convention there, since it's the name of the GCC builtin. I deally these functions would have a bit more intuitive names. Second, I take it this is short for "find last zero". It's not entirely obvious which bit is "last", is it the most significant or the least significant one? Also, at least bit indices are well established to start at 0 from the least significant bit. So I take it this returns the bit index of the last zero, is that right?